src/Title/Title.scss

```diff
@@ -4,3 +4,23 @@
     font-size: 50px;
     color: $primary;
 }
+
+.landing-page-title, .catch-phrase{
+    &.show-restaurants {
+        display: block;
+    }
+
+    &.show-menu {
+        display: none;
+    }    
+}
+
+.title-restaurant-name {
+    &.show-restaurants {
+        display: none;
+    }
+
+    &.show-menu {
+        display: block;
+    }    
+}
```

Comment:
as a result of my previous comment all this code will probably disappear

Reply:
I've removed it